source/common/upstream/health_checker_impl.cc

```diff
@@ -195,8 +195,8 @@ void HttpHealthCheckerImpl::HttpActiveHealthCheckSession::onInterval() {
     expect_reset_ = false;
   }
 
-  request_encoder_ = &client_->newStream(*this);
-  request_encoder_->getStream().addCallbacks(*this);
+  Http::StreamEncoder* request_encoder = &client_->newStream(*this);
```

Comment:
Is this just a cleanup or does is this part of the fix?

Reply:
It's just a cleanup. The member variable was not used so I deleted it.